src/mscorlib/src/System/Exception.cs

```diff
@@ -748,7 +656,7 @@ internal void RestoreExceptionDispatchInfo(System.Runtime.ExceptionServices.Exce
 
         private String _className;  //Needed for serialization.  
         private MethodBase _exceptionMethod;  //Needed for serialization.  
-        private String _exceptionMethodString; //Needed for serialization. 
+        private String _exceptionMethodString; //Needed for serialization.
```

Comment:
That's a little too risky to include in this change. I've filed https://github.com/dotnet/coreclr/issues/11906 to track removing the field.